### PR TITLE
Capston project: Budget-App

### PR DESCRIPTION
**Pull request by ForHemer**
The Ruby on Rails capstone project is about building a mobile web application where users can manage their budget: They have a list of transactions associated with a category, so that they can see how much money they spent and on what.

The **Budget-App** allows the user to:

- [ ] register and log in, so that the data is private to them.
- [ ]  introduce new transactions associated with a category.
- [ ] see the money spent on each category.

For more details, see [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md).

### CODE REVIEW

File: spec/models/group_spec.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      group.name = 'qwertyuiopasdfghjklzxcvbnm,qwertyuiopasdfghjklzxcvbnmqwertyuiopasdfghj,qwertyuiopasdfghjklzxcvbnm,,qwertyuiopasdfghjklzxcvbnm,'
+      expect(group).to_not be_valid
+    end
+
+    it 'should allow valid user_id' do
+      group.user_id = nil
+      expect(group).to_not be_valid
+    end
+  end
+end
```
Comment (rotimiazeez) [inline spec/models/group_spec.rb]: - Kindly make sure all tests are passing 😉 
-
Screenshot from 2022-07-01 03-40-52

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @ForHemer 👋🏽, 
## Good job so far! 👍🏽 
### Quick summary 🔥 
- Great video presentation 👏🏽
- Good and descriptive PR 👍🏽 
- Good commit messages 😉 
- Well documented README 😃 
- No Linter Errors ✔️ 
- Good clean code 👍🏽 
- App deployed 💯
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/group_spec.rb (hunk 1 above)
Comment (ForHemer) [reply spec/models/group_spec.rb]: @rotimiazeez Thank you! Everything is correct now!

**Review by OLIPLICHE — APPROVED** [review]
Hi @ForHemer , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**